apps/trend-rights/pages/why-we-are/index.tsx

```diff
@@ -38,9 +39,12 @@ export const getServerSideProps = async (
   const locale = context.locale as StrapiLocale
   const { token } = await getSession(context.req, context.res)
 
+  const currentCategory =
+    (context.query.category as string) || BLOG_CATEGORIES.DEFAULT
+
   await queryClient.prefetchQuery({
     queryKey: ['blogs', locale],
```

Comment:
QueryKey should be the same as client to make hydration work properly.

```suggestion
    queryKey: ['blogs', locale, currentCategory],
```